apps/central/src/components/SchemaCreateModal.vue

```diff
@@ -178,7 +179,7 @@ export default {
       ) {
         return undefined;
       } else {
-        return "Table name must start with a letter, followed by letters, underscores, a space or numbers, i.e. [a-zA-Z][a-zA-Z0-9_]*. Maximum length: 31 characters";
+        return "Table name must start with a letter, followed by letters/underscores/spaces/numbers (though no underscore preceded/followed by a space) and with a maximum of 31 characters, i.e. ^(?!.* _|.*_ )[a-zA-Z][a-zA-Z0-9 _]{0,30}$";
```

Comment:
Should we put the regex pattern in the error message? It is helpful for developers, but I'm not sure if many end users (data managers, lab personnel, etc.) would understand this.

Reply:
Fair point. I left it in as they were present already, though for a user (without IT background) it indeed can just cause confusion.